sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -1147,6 +1147,18 @@ public interface OptionsConfiguration<T extends SentryOptions> {
     return getCurrentScopes().continueTrace(sentryTrace, baggageHeaders);
   }
 
+  /**
+   * Continue a trace based on the trace ID and span ID provided
+   *
+   * @param traceId the trace ID
+   * @param spanId the span ID
+   */
+  // return TransactionContext (if performance enabled) or null (if performance disabled)
+  public static void setTrace(
+      final @NotNull String traceId, final @NotNull String spanId) {
```

Comment:
Based on the `sample_rand` spec. https://github.com/getsentry/sentry-docs/pull/12592

The continuing of a trace should require `sample_rand`.

Reply:
I've moved `setTrace` to `InternalSentrySdk` and it additionally accepts a `sample rate` and `sample rand`.